draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -6045,3 +6045,818 @@ Q.y     = 1260d6e0987eae96af9ebe551e08de22b37791d53f4db9e0d59da7
     + I * 0dbace5df0a4ac4ac2f45d8fdf8aee45484576fdd6efc4f98ab9b9
           f4112309e628255e183022d98ea5ed6e47ca00306c
 ~~~
+
+# Expand test vectors {#expand-testvectors}
+
+This section gives test vectors for expand_message variants specified in {{hashtofield-expand}}.
+Each test vector in this section lists the expand_message name, hash, DST, and
+security parameter, along with a series of tuples of the function inputs (msg),
+outputs (pseudo_random_bytes), and intermediary values (dst_prime and msg_prime).
+Byte strings are encoded as hexadecimal strings where appropriate.
+
+## expand_message_xmd(SHA-256)
+
+name    = expand_message_xmd
+dst     = QUUX-V01-CS02-with-expander
+hash    = SHA256
+security_param = 128
+
+dst_prime = 515555582d5630312d435330322d776974682d657870616e646572
```

Comment:
what is the input msg?

Reply:
Oh whoops -- I added it to the dictionary but forgot to render it. Stand by!